main/ui/src/main/java/org/cryptomator/ui/recoverykey/RecoveryKeyCreationController.java

```diff
@@ -59,6 +64,7 @@ public void createRecoveryKey() {
 				Animations.createShakeWindowAnimation(window).play();
 			} else {
 				LOG.error("Creation of recovery key failed.", task.getException());
+				errorComponent.cause(task.getException()).window(window).returnToScene(createScene.get()).build().showErrorScene();
```

Comment:
Also here applies the same:
An alternative would be to get the current scene over the window: `window.getScene()` The benefit would be in reducing code (variable definitions and object injections)

Reply:
Resolved by d331c3022489c7d0651a588e0c58f877064d55a8.